### DISCUSSION TITLE
FlinkCDC Out of memory

### DISCUSSION THREAD

**Discussion by gilboom** [apache/doris#22439] [A - General / Q&A]
当我使用 FlinkCDC 以 Flink SQL 的方式去将 MySQL 数据导入到 Doris 的时候，发现内存的占用会慢慢的增大，最终 Out Of Memory。

我已经使用了 EmbeddedRocksDB 作为 Flink 的状态后端了，但发现还是会占用大量的内存。

connector 版本：1.17-1.4.0
doris 版本：1.2.4-1

**Comment by LemonLiTree** [comment]
flink OOM or doris OOM?

**Reply by gilboom** [reply]
Flink OOM，我想我可能排查问题出来了，JobManager 设置内存的太大了，TaskManager 设置的内存太小了

**Reply by LemonLiTree** [reply]
Great
